views/css/item-comments.css

```
@@ -1,270 +1,403 @@
+.item-comment .item-comment-meta {
+  font-size: 12px;
+  color: #666;
+  margin-bottom: 4px;
+}
+.item-comment .item-comment-author {
+  font-weight: 600;
+  color: #333;
+}
+.item-comment .item-comment-body {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)